src/ngx_http_lua_common.h

```diff
@@ -261,12 +275,8 @@ typedef struct {
 
     ngx_http_output_body_filter_pt         body_filter_handler;
 
-    u_char                  *rewrite_chunkname;
-    ngx_http_complex_value_t rewrite_src;    /*  rewrite_by_lua
-                                                inline script/script
-                                                file path */
-
-    u_char                  *rewrite_src_key; /* cached key for rewrite_src */
+    ngx_array_t             *rewrites;  /*   rewrite sets, inline script/script
```

Comment:
There are `rewrite_handler` and `rewrite_handlers` in struct `ngx_http_lua_loc_conf_t`.  Will it cause any confusion?